samples/csharp_dotnetcore/54.teams-task-module/TeamsAppManifest/manifest.json

```diff
@@ -11,8 +11,8 @@
     "termsOfUseUrl": "https://example.azurewebsites.net/termsofuse"
   },
   "icons": {
-    "color": "color.png",
-    "outline": "outline.png"
+    "color": "icon-color.png",
+    "outline": "icon-outline.png"
```

Comment:
Is this change necessary?

Reply:
It is. Otherwise, the manifest can't be uploaded in Teams.